package.json

```diff
@@ -4,7 +4,7 @@
   "main": "index.js",
   "scripts": {
     "start": "cross-env MODE=development CONFIG=as-development webpack serve --open",
-    "build": "cross-env webpack",
+    "build": "cross-env MODE=development CONFIG=as-development webpack",
```

Comment:
This is more a question. I am not able to build without the configurations. Is it fair to specify this? Is there a reason to remove `build:dev`